rpcs3/rpcs3qt/pad_settings_dialog.cpp

```diff
@@ -909,6 +917,7 @@ void pad_settings_dialog::ChangeInputType()
 	bool force_enable = false; // enable configs even with disconnected devices
 	const int player = m_tabs->currentIndex();
 
+
```

Comment:
What's this line for

Reply:
Leftover from some other thing, I somehow missed it when going through the diff.